docs/v0.0.1/layout-specs.md

```diff
@@ -41,90 +82,182 @@ Each element in the hierarchical structure contains the following key-value pair
-      "elementType": "category",
-      "include": [
-        { "path": "DSCore.Color" },
-        { "path": "DSCore.ColorRange2D" }
-      ],
-      "childElements": []
+      "text": "default",
+      "iconUrl": "",
+      "elementType": "section",
+      "showHeader": false,
+      "include": [],
+      "childElements": [
+        {
+          "text": "Display",
+          "iconUrl": "/src/resources/icons/Category.Display.svg",
+          "elementType": "category",
+          "include": [
+            {
+              "path": "DSCore.Color",
+              "iconUrl": "/src/resources/icons/DSCore.Color.png"
+            },
+            {
+              "path": "DSCore.ColorRange",
+              "iconUrl": "/src/resources/icons/DSCore.ColorRange.png"
+            }
+          ],
+          "childElements": []
+        }]
     }]
 }
 ```
+Note that this will create two elements with *none* as `elementType`, named "Color" and "ColorRange", each containing nodes with names that start with "DSCore.Color" and "DSCore.ColorRange" respectively:
 
-This results in library view that looks like the following image:
-
-![image](img/layout-include.png?raw=true)
+![image](img/layout-include-class.png?raw=true)
 
 ### Adding nested elements
 Adding on to the previous example, a nested element with text `Watch` is added under `Display` element. Just like the parent element, nested elements contain keys like `text`, `iconUrl`, etc. Note that however, the nested element does not have to contain `childElements` if it does not need to:
 
 ```json
 {
-  "elements": [
+  "sections": [
     {
-      "text": "Display",
-      "iconUrl": "Category.Display.svg",
-      "elementType": "category",
-      "include": [
-        { "path": "DSCore.Color" },
-        { "path": "DSCore.ColorRange2D" }
-      ],
+      "text": "default",
+      "iconUrl": "",
+      "elementType": "section",
+      "showHeader": false,
+      "include": [],
       "childElements": [
         {
-          "text": "Watch",
-          "iconUrl": "Watch",
-          "elementType": "none",
-          "include": []
-        }        
-      ]
+          "text": "Display",
+          "iconUrl": "/src/resources/icons/Category.Display.svg",
+          "elementType": "category",
+          "include": [
+            {
+              "path": "DSCore.Color",
+              "iconUrl": "/src/resources/icons/DSCore.Color.png"
+            },
+            {
+              "path": "DSCore.ColorRange",
+              "iconUrl": "/src/resources/icons/DSCore.ColorRange.png"
+            }
+          ],
+          "childElements": [
+            {
+              "text": "Watch",
+              "iconUrl": "Watch",
+              "elementType": "group",
+              "include": []
+            }   
+          ]
+        }]
     }]
 }
 ```
+Since `Watch` does not have any child items, it will not be rendered in the library. For the next step, we will add some leaf nodes into `Watch`.
```

Comment:
Maybe we can avoid using terms like *we* in this documentation. I would suggest the following rephrase:

*Since `Watch` element does not have any child item (i.e. its `include` list is empty), it will not be rendered in the library. The next section illustrates how leaf items are added under `Watch` so that it shows up on screen.*